src/controllers/expenseController.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable no-console */
+'use strict';
+
+const { ExpenseService } = require('../services/expenseService');
+const expenseService = new ExpenseService();
+const { Op } = require('sequelize');
+const userService = require('../services/userService');
+
+async function createExpense(req, res) {
+  try {
+    const { userId, spentAt, title, amount, category, note } = req.body;
+
+    if (!spentAt || !title || !amount || !userId) {
+      return res.status(400).send('All fields are required');
+    }
+
+    const user = await userService.getAllUsers({ id: userId });
```

Comment:
Not super needed, but `userService` could have a function called `validadeUser` or something on this matter, that is called to check if that user exists and returns a boolean.
Because you use this test here in multiple places, would be a good idea.